ScanCommand/src/main/java/io/ballerina/scan/InternalReporter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan;
+
+import io.ballerina.projects.Document;
+import io.ballerina.tools.diagnostics.Location;
+
+import java.util.List;
+
+class InternalReporter {
```

Comment:
Should we include a brief description (even just one line would suffice) along with the `@since` annotation to specify in which version it was introduced? Apply this for other classes as well

Reply:
Applied with [f7de1b9](https://github.com/ballerina-platform/static-code-analysis-tool/pull/4/commits/f7de1b9af84e262f93ade17a1643ffb9f9b64687)